concourse/templates/web-deployment.yaml

```diff
@@ -5,22 +5,29 @@ metadata:
   labels:
     app: {{ template "web.fullname" . }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: web
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
```

Comment:
You will need to specify the serviceaccount you created if you want to use it. (I don't think concourse needs serviceaccount/rbac)